src/Parser/HolyPriest/CombatLogParser.js

```diff
@@ -0,0 +1,271 @@
+  if (higherIsWorse ? value > major : value < major) {
+    return ISSUE_IMPORTANCE.MAJOR;
+  }
+  if (higherIsWorse ? value > regular : value < regular) {
+    return ISSUE_IMPORTANCE.REGULAR;
+  }
+  return ISSUE_IMPORTANCE.MINOR;
+}
+
+function formatPercentage(percentage) {
+  return (Math.round((percentage || 0) * 10000) / 100).toFixed(2);
+}
+
+class CombatLogParser extends MainCombatLogParser {
+  static abilitiesAffectedByHealingIncreases = ABILITIES_AFFECTED_BY_HEALING_INCREASES;
+
+  static specModules = {
+    abilityTracker: AbilityTracker,
+
+    // Features
+    alwaysBeCasting: AlwaysBeCasting,
+    cooldownTracker: CooldownTracker,
+    renewTheFaith: RenewTheFaith,
+
+    // Spells
+    prayerOfMending: PrayerOfMending,
+    divineHymn: DivineHymn,
+
+    // Items
+    trousersOfAnjuna: TrousersOfAnjuna,
+    xanshiCloak: XanshiCloak,
+  };
+
+  generateResults() {
+    const results = super.generateResults();
+
+    const fightDuration = this.fightDuration;
+    const deadTimePercentage = this.modules.alwaysBeCasting.totalTimeWasted / fightDuration;
+
+    const abilityTracker = this.modules.abilityTracker;
+    const getAbility = spellId => abilityTracker.getAbility(spellId);
+
+    const castEfficiencyCategories = SPELL_CATEGORY;
+    const castEfficiency = getCastEfficiency(CPM_ABILITIES, this);
+
+    // Renew the Faith trait data calculations
+    const rtfPercHPS = formatPercentage(this.modules.renewTheFaith.healing / this.totalHealing);
+    const rtfPercOH = formatPercentage(this.modules.renewTheFaith.overhealing / (this.modules.renewTheFaith.healing + this.modules.renewTheFaith.overhealing));
+    const rtfRelPercHPS = formatPercentage(this.modules.renewTheFaith.healing / this.modules.divineHymn.healing);
+
+    // Say Your Prayers trait data calculations
+    const averagePoMperCast = (this.modules.prayerOfMending.removed - this.modules.renewTheFaith.poms) / getAbility(SPELLS.PRAYER_OF_MENDING_CAST.id).casts;
+    const sypValue = (averagePoMperCast - 5) / averagePoMperCast * this.modules.prayerOfMending.healing;
+    const sypHPS = sypValue / this.fightDuration * 1000;
+    const sypPercHPSOverall = formatPercentage(sypValue / this.totalHealing);
+    const sypPercHPSPoM = formatPercentage(sypValue / this.modules.prayerOfMending.healing);
+
+    // Missed Hymn ticks calculation
+    const missedHymnTicks = (getAbility(SPELLS.DIVINE_HYMN_CAST.id).casts * 5) - this.modules.divineHymn.ticks;
+
+    // Leggo Legs calculations
+    const legsPercHPS = formatPercentage(this.modules.trousersOfAnjuna.healing / this.totalHealing);
+    const legsHPS = formatNumber(this.modules.trousersOfAnjuna.healing / this.fightDuration * 1000);
+
+    // Leggo cloak (Xan'shi) calculations
+    const cloakPercHPS = formatPercentage(this.modules.xanshiCloak.healing / this.totalHealing);
+    const cloakHPS = formatNumber(this.modules.xanshiCloak.healing / this.fightDuration * 1000);
+
+    if (deadTimePercentage > 0.1) {
```

Comment:
You might also want to check for non healing time, which generally has stricter suggestions (for healers). Using your time to deal damage usually isn't nearly as important as using it to heal - as a healer.